src/protocols/secure_channel/CASESession.cpp

```diff
@@ -45,54 +45,69 @@
 #include <protocols/secure_channel/SessionResumptionStorage.h>
 #include <protocols/secure_channel/StatusReport.h>
 #include <system/SystemClock.h>
-#include <system/TLVPacketBufferBackingStore.h>
 #include <tracing/macros.h>
 #include <tracing/metric_event.h>
 #include <transport/SessionManager.h>
 
 namespace {
 
-enum
+enum class Sigma1Tags : uint8_t
 {
-    kTag_TBEData_SenderNOC    = 1,
-    kTag_TBEData_SenderICAC   = 2,
-    kTag_TBEData_Signature    = 3,
-    kTag_TBEData_ResumptionID = 4,
```

Comment:
Please re-order so that the diff is more correct. Please leave TBE with TBE, TBS with TBS, positionally in the file.

Reply:
Done